open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+  Don't go to that dumb party. Do something fun instead!
+---
+id: go_advice
+event: go_to_party
 question: |
-  Are you at least 18 years of age?
-yesno: over_eighteen
+  You should go to the party! I promise it'll be awesome!
+---
 ```
 
-And here's what it looks like when we run it:
-
-<img alt="Yes/No question" src={useBaseUrl('img/Yes_no.png')} />
+### What do you need to know?
 
-Docassemble uses two different kinds of `question` blocks. This new kind only lets you ask for one variable on a screen, and doesn't use the `fields` specifier we learned earlier. It's kind of a simplified shorcut version.
+Well, without user research, you don't really know what good questions would be. You don't have funding or time to do that discovery work, though, but you don't care that much about those strangers and it's not like they have other options, so you'll start by making a guess and worry about it later.
 
-Let's look at line 3 of our sample:
+Do they even like parties? That seems like a place to start.
 
 ```yaml
-yesno: over_eighteen
+---
+id: does_like_parties
+question: |
+  Do you like parties?
+fields:
+  - Do you like parties: user_likes_parties
+---
 ```
+<!-- I'd stick with fields for now because those will be used most consistently -->
 
-This is a new specifier: the `yesno` specifier. It always shows the text `yes` and `no` as buttons on the screen. On the right hand side of the : is the variable that will be assigned the value `True` or `False`, depending on which button the user clicks.
-
-### Planning out our variables
-
-It can help to start out by writing a list of the actual variables we will want to ask. This way, we can plan variable names that make sense together.
-
-In Docassemble interviews, it's traditional to use variables names that are:
+Docassemble will show two buttons on this question screen. If the user clicks 'Yes', `user_likes_parties` will have a value of `True`. If they click 'No', it will have a value of `False`. We can use that to help our program decide what to do. [Read more about yes/no fields](https://docassemble.org/docs/fields.html#fields%20yesno).
 
-1. Short
-1. Descriptive (without cryptic abbreviations)
-1. Lower-case
-1. Use _ to separate multiple words when needed
+### If this, then that
 
-We are going to need true/false variables and an ending screen. Let's name them as follows:
+We'll use what's called an `if` statement. It tells our program, 'if the situation is this, then do that.'
 
-variable name | meaning
---------------|---------
-go_to_party | A screen that tells the person to go to the party
-dont_go_to_party | A screen that says not to go to the party
-likes_parties | Do you like parties?
-wants_to_meet | Do you want to meet someone there?
-big_assignment | Do you have a big assignment due?
-
-### Writing our first question blocks
-
-Let's start with writing the ending blocks, which are the simplest. Remember, we will use the `event` specifier for these because they are ending screens.
```

Comment:
Note: Maybe an example of a good spot for linking to documentation for the `event` attribute here. You do a much better job of linking to documentation than I do in general. Your style contributed to the links I did add. Also, is 'specifier' its technical name?

Reply:
I'm using `specifier` consistently in class. It's used in most places in the Docassemble documentation, but it's a term specific to Docassemble. Credit to Everett from Clerical.ai

Reply:
What's it called in yaml?